client/public/index.html

```diff
@@ -28,6 +28,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBrthKEyiw7uHg8FXRFcpunPKlKqxY1z5k&libraries=places"></script>
```

Comment:
Unfortunately there's no npm package for the Google Maps Places API, this is the only way to include it in the project.